ui/src/app/utils/access-utils.tsx

```diff
@@ -314,6 +339,27 @@ export const getAccessModuleConfig = (
   );
 };
 
+export const getInitialRTModules = (): AccessModule[] => {
```

Comment:
Moved these from static lists to functions in order to make use of feature flagging.

Reply:
`getInitialRTModules` is strictly used for display logic for the data access requirements page. 

This function helps determines which cards are shown. The DARTitleComponent of Login.gov shows the current identity verification card, but the DARTitleComponent of RASLINKIDME shows the new design where both identity providers are represented. We would not want both of these cards to shown on the same screen, because they are redundant. That is why the Login.gov card is not shown when ID.me is active.

As I am writing this though, I am wondering if it would be more intuitive to always show RASLINKLOGINGOV (never show RASLINKIDME), and make the DARTitleComponent conditionally render. Show the old design when ID.me is not active and show the new design when ID.me is enabled.

Does one approach make more sense than the other to you @Peter-Lavigne?

Reply:
> is strictly used for display logic for the data access requirements page

Maybe the function should be renamed to indicate this?

> it would be more intuitive to always show RASLINKLOGINGOV (never show RASLINKIDME), and make the DARTitleComponent conditionally render

I think that would be clearer since that'll be the end result once the `enableRasIdMeLinking` feature flag is removed (if this isn't a FF, please let me know).

Maybe there's a way to refactor this to return single concept representing both access modules (like a list containing both) so that it's clear both access modules are represented by one downstream component.

Reply:
Switching the component rendering logic to a single module made the code simpler, because it means that the feature flag is not needed when deciding which modules to show. This means that I could switch the module lists to lists instead of functions that return lists.